Improve Azure log collection docs (#26806)
DataDog/documentation: content/en/continuous_integration/pipelines/azure.md

1. Install a Datadog app registration on your Azure console. Follow the steps in the [Azure integration tile][14].

> 2. Add the Datadog app registration to your Azure DevOps organization. Navigate to **Organization settings** in your DevOps console, click on **Users** on the left side panel, then click on **Add Users**. If you don't see the button, you may not have the necessary permissions. You should add your app registration as a user with Basic Access Level to each project for which you want to enable log collection. Alternatively, you can click on  **Add to all projects** to configure all projects in bulk.

Reviewer: ```suggestion
2. Add the Datadog app registration to your Azure DevOps organization:
  a. Navigate to **Organization settings** in your DevOps console.
  b. Click **Users** from the left side panel, then click on **Add Users**.
  **Note**: If you don't see the **Add Users** button, you may not have the necessary permissions.

To enable log collection, add your app registration as a user with Basic Access Level to each project. Alternatively, you can click  **Add to all projects** to configure all projects in bulk.
```
